### DISCUSSION TITLE
Transfer Settings to Another Computer?

### DISCUSSION THREAD

**Discussion by SoCalDude** [martinet101/ElevenClock#1177] [Q&A]
I searched here on this repo's Github discussions, but did not find an answer, so here is my question...

**_How can I copy my settings on one computer to another computer?_** The settings appear to be stored in:

`%HOMEPATH%\.elevenclock`

Is it just a matter of copying that folder to another computer under that destination's user's home path?

**Comment by marticliment** [comment]
yes, you can literally do that, or you can also export your settings from ElevenClock settings (about ElevenClock - > export preferences) and then import them on another machine
